tests/test_lib.py

```diff
@@ -257,6 +257,93 @@ def test_create_workspace(self, requests_mock):
+                    "previous": None,
+                    "next": f"{ANNOTATIONS_URL}?queue=0&status=exported&&sideload=documents&page=2",
+                },
+                "results": [
+                    {
+                        "id": 0,
+                        "url": f"{ANNOTATIONS_URL}/0",
+                        "queue": f"{QUEUES_URL}/0",
+                        "document": f"{DOCUMENTS_URL}/0",
+                        "status": "exported",
+                    }
+                ],
+                "documents": [{"id": 0, "url": f"{DOCUMENTS_URL}/0"}],
+            },
+            f"{ANNOTATIONS_URL}?queue=0&status=exported&&sideload=documents&page=2": {
+                "pagination": {
+                    "total": 3,
+                    "total_pages": 2,
+                    "previous": f"{ANNOTATIONS_URL}?queue=0&status=exported&&sideload=documents",
+                    "next": None,
+                },
+                "results": [
+                    {
+                        "id": 1,
+                        "url": f"{ANNOTATIONS_URL}/1",
+                        "queue": f"{QUEUES_URL}/0",
+                        "document": f"{DOCUMENTS_URL}/1",
+                        "status": "exported",
+                    },
+                    {
+                        "id": 2,
+                        "url": f"{ANNOTATIONS_URL}/2",
+                        "queue": f"{QUEUES_URL}/0",
+                        "document": f"{DOCUMENTS_URL}/2",
+                        "status": "exported",
+                    },
+                ],
+                "documents": [
+                    {"id": 1, "url": f"{DOCUMENTS_URL}/1"},
+                    {"id": 2, "url": f"{DOCUMENTS_URL}/2"},
+                ],
+            },
+        }
+
+        expected = [
+            {
+                "id": 0,
+                "url": f"{ANNOTATIONS_URL}/0",
+                "queue": f"{QUEUES_URL}/0",
+                "document": {"id": 0, "url": f"{DOCUMENTS_URL}/0"},
+                "status": "exported",
+            },
+            {
+                "id": 1,
+                "url": f"{ANNOTATIONS_URL}/1",
+                "queue": f"{QUEUES_URL}/0",
+                "document": {"id": 1, "url": f"{DOCUMENTS_URL}/1"},
+                "status": "exported",
+            },
+            {
+                "id": 2,
+                "url": f"{ANNOTATIONS_URL}/2",
+                "queue": f"{QUEUES_URL}/0",
+                "document": {"id": 2, "url": f"{DOCUMENTS_URL}/2"},
+                "status": "exported",
+            },
+        ]
+
+        for mock_url, mock_data in api_response_pages.items():
+            requests_mock.get(
+                mock_url,
+                request_headers={"Authorization": f"Token {TOKEN}"},
+                json=mock_data,
+                status_code=200,
+            )
+
+        assert expected == self.api_client.get_annotations(
+            queue=0, status=["exported"], sideloads=[DOCUMENTS]
```

Comment:
Consider using `QUEUE_ID` constant defined at the top of this module instead of `0` and applying similar principle to the `DOCUMENT_ID` (and `DOCUMENT_URL`, respectively) used above